docs/packages/margarita-components/index.md

```diff
@@ -3,6 +3,13 @@ title: Margarita Components
 sidebar_label: Margarita Components
 ---
 
-_This page is a stub._
+Margarita Components is a local package to this repository, which contains components that are specific and tailored to the margarita needs and business logic, by composing components from `@kiwicom/universal-components`.
```

Comment:
Maybe we can link `@kiwicom/universal-components` to `./universal-components`.
Otherwise looks good.

Reply:
Good idea. Done!